Makefile

```diff
@@ -26,10 +32,15 @@ build: stop setup
 
 serve: build
 	docker-compose up -d
+	./scripts/wait_for_apps
+	docker-compose exec service-token-cache-app sh -c "rails runner \"Support::ServiceTokenCache.put('slug', 'token-for-slug')\""
```

Comment:
Nice use of local stack.
Wonder if within the context of this project, we should name it something like fake-s3?
Not too fussed.

Also are we able to prepend all these commands as a 'command:' instruction in the docker compose file, just before we boot it up?

Reply:
i can rename to `fake-s3`. localstack seems to support SES too, but we can deal with that if and when we get there.

i will extract out the commands

Reply:
:+1: for getting this all into docker compose